checker.js

```diff
@@ -0,0 +1,279 @@
+/**
+ * Created by timmimim on 19.04.17.
+ */
+
+var fs = require('fs');
+var path = require('path');
+var exec = require('child_process').exec;
+
+var iterate;
+
+// RegEx to find all img tags
+var allImgTagsAsStrings = /<img src="data:image\/png;base64,(.*)" \/>/g;
+var regexSplitCuttingImages = /<img src="data:image\/png;base64,.*" \/>/g;
+
+
+/* TODO: splice HTML into head and body, only then extract base64-images from body and then compare remaining snippets textually;
+   TODO: for now: leave Head with first body-segment, split HTML String at every Image, cutting out images and save to array "splitHTMLexcludingImages"*/
+//var head = /regex/
+var arrayOriginalHTMLexcludingImages;
+var arrayReproducedHTMLexcludingImages;
+
+var boolArrayImageDiffOrdered = [];
+
+
+function stringifyHTMLandCompare(originalPaperHTML, reproducedPaperHTML, outputName) {
+	fs.readFile(originalPaperHTML, 'utf-8', function (err, dataOriginal) {
+		if (err) {
+			console.log("  Unable to read the first (original) file as String. Something has gone wrong. \n \n  Maybe check your input path.");
```

Comment:
Also, you must at least print out `err.message` to help the user finding the problem!